tests/functional/test_k4_world_clock_cities.py

```
@@ -30,10 +30,37 @@ def test_city_count_and_plate_entries_differ_by_one(self):
+        assert hours["KAMTSCHATKA"] == hours["MAGADAN_SACHALIN"] + 1
+        assert hours["KAPDESCHNEW"] == hours["KAMTSCHATKA"] + 1
+        assert hours["HONOLULU"] == hours["KAPDESCHNEW"] + 1
+
+
+class TestRunWorldClockSectorSweep:
+    def test_null_result_artifact(self, tmp_path):
+        artifact = tmp_path / "sector_null.json"
+        summary = wcc.run_world_clock_sector_sweep(null_artifact_path=str(artifact))
+        assert summary["status"] == "null_result"
+        data = json.loads(artifact.read_text(encoding="utf-8"))
+        assert set(data["run_params"]["rotation_offsets"]) == set(wcc.world_clock_sector_offsets().values())
```

Pylint (4.0.7)

[convention] Line too long (108/88)

(C0301)